src/app/src/test/kotlin/org/imperial/mrc/hint/integration/IntegrationTests.kt

```diff
@@ -68,7 +68,7 @@ abstract class SecureIntegrationTests : CleanDatabaseTests() {
         when (isAuthorized) {
             IsAuthorized.TRUE -> {
                 Assertions.assertThat(responseEntity.headers.contentType!!.toString())
-                        .isEqualTo("application/json")
+                        .contains("application/json")
```

Comment:
When Spring wraps up the response entity it adds the charset UTF-8 bit to the end, unlike hintr, so just making this permissive of both.